WordPress/build.gradle

```diff
@@ -109,11 +111,13 @@ android {
         wasabi { // "hot" version, can be installed along release, alpha or beta versions
             applicationId "org.wordpress.android.beta"
             dimension "buildType"
+            buildConfigField "boolean", "MODAL_LAYOUT_PICKER", "true"
```

Comment:
Hey @antonis, was coming back to this to check on the progress. This is something I didn't notice while scoping out the issue, but I wonder if this should be enabled under `zalpha` instead of under the `wasabi` and `jalapeno` based on the comment `// alpha version - enable experimental features`

I didn't do a full review yet but wanted to call out we might want to check with someone from the main WPAndroid teams to double-check.

Reply:
Hello @chipsnyder 👋 ,
I initially enabled this for [`vanilla` and `jalapeno`](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2419) but after talking to @pinarol I changed it to `wasabi` and `jalapeno` since we did not want this enabled in beta yet. I cross checked this configuration with @malinajirka last week.
To my understanding we can enable this in `zalpha` too.

Reply:
Just to summarise. It really depends on who should have access to this feature.

- `wasabi` is for developers
- `jalapeno` is for reviewers (automatic CI builds on PRs - needs to be manually downloaded and installed)
- `zalpha` is for alpha testers (closed track distributed through Google Play)
- `vanilla` is for beta testers and users/release (open track + release distributed through Google Play)

Reply:
That's more clear now then. Thank you both for the explanation 🙂